### DISCUSSION TITLE
Consiglio mini PC per HA

### DISCUSSION THREAD

**Discussion by lucaxxaa** [indomus/forum#1155] [Hardware & Firmware]
Buongiorno.
Mi consigliate un mini PC per HA?

Preferirei che sia fanless, con il miglior rapporto qualità/prezzo

Non devo fare particolari cose, ma attualmente con Raspberry pi4 con SSD ho diversi blocchi del sistema.
Attualmente uso hassio, ma eventualmente su mini PC, installerei proxmox per provare altro
Grazie

**Comment by KC-inDomus** [comment]
Perché piuttosto di spendere inutilmente dei soldi non smetti di usare hassio e cominci ad usare Core su Docker?
Sono pronto a scommettere una birra che non avrai più alcun tipo di problema, non dar retta a chi ti dice il contrario. Mente, per ignoranza o tifoseria.

**Reply by lucaxxaa** [reply]
Uso Rpi4 con boot su SD e SSD esterno.
Installare core su docker, vuol dire usare una sistema operativo a se tipo raspian e poi procedere ad installare il sever virtuale?

In questo caso, che distro dovrei usare? ( raspian )  potrei usare direttamente SSD senza microsd?
Grazie

**Reply by KC-inDomus** [reply]
guarda [qui](https://indomus.it/formazione/home-assistant-su-raspberry-pi-varie-installazioni-le-cose-da-fare/), hai tutte le cose da fare, in ordine (sezione Core su Docker). sì, volendo puoi anche installare raspbian direttamente su ssd (la guida per installare raspbian "headless" spiega come fare).

**Reply by lucaxxaa** [reply]
grazie. 

Ci provero con calma

**Reply by KC-inDomus** [reply]
e di che. se hai problemi affacciati in chat telegram 👍

**Reply by giovannicaliendo** [reply]
Ciao, se vuoi prendere in considerazione altro hw, io utilizzo un mini PC della Asrock, basato sulla piattaforma Intel NUC, con processore Celeron 3000, completamente fanless, con 8 GB di RAM e disco SSD M2-sata. Ho scollegato dal PC la schedina del wifi-bluetooth perché utilizzo esclusivamente il cavo LAN. Sopra ho installato Ubuntu 18.04 LTS e Docker: su container ho messo HA, il broker MQTT, Node-red, Pihole e Wireguard (che utilizzo come server VPN per accedre da remoto alla mia rete); tutti container sono gestiti da Docker-compose. E' acceso H24 ed è stabile come una roccia.

**Reply by snakuzzo** [reply]
@giovannicaliendo con questa configurazione, salvo eventuale utilizzo di diverse telecamere (con transcodifica) o in generale cose che richiedono utilizzo massivo di CPU, ti basta anche un RPI3B.
Fino a poco tempo fa usavo un RPI3B con tutto l'occorrente su Docker (Core con 2 cam senza transcodifica + Mosquitto + Caddy + JDownloader + Emby Media Server + Node-Red + Adguard + OpenVPN Server + Esphome + Diun + Portainer) e non ho mai avuto problemi...h24 7gg/7gg ...mai avuto necessità di riavviare,
Direi che, in ottica dismissione HassOS in favore di applicazioni dockerizzate, il consiglio resta quello di rimanere su RPI4 (che è fin troppo performante)

**Reply by giovannicaliendo** [reply]
@snakuzzo è vero quello che dici, il mini PC forse è eccessivo. In realtà utilizzo il RPI3B con il suo schermo touch come chiosco per gestire HA: purtroppo, periodicamente l'RPI3B perde la connettività di rete (lo uso esclusivamente via cavo) e mi tocca riavviarlo. Siccome sono sensibile al discorso resilienza visto che su HA gestisco la climatizzazione di casa e la rilevazione movimento nei box tramite le telecamere, trovo il mini PC molto più affidabile dell'RPI3B.....(poi magari scopro che il mio RPI3B è difettoso....)

**Reply by snakuzzo** [reply]
Credo sia difettoso...perché io questo problema non l'ho mai avuto

EDIT: o magari potrebbe esserci qualcosa a livello di OS che causa questo problema. Toccherebbe capire cosa accade andando ad analizzare il syslog ad esempio
